### PR TITLE
fix(contracts): init

### PR DESCRIPTION
**Pull request by ratankaliani**

### CODE REVIEW

File: contracts/src/BlobstreamX.sol [hunk 1]
```diff
@@ -24,10 +24,10 @@ contract BlobstreamX is
 
     /// @notice The maximum number of blocks that can be skipped in a single request.
     /// Source: https://github.com/celestiaorg/celestia-core/blob/main/pkg/consts/consts.go#L43-L44
-    uint64 public DATA_COMMITMENT_MAX = 1000;
+    uint64 public constant DATA_COMMITMENT_MAX = 1000;
 
     /// @notice Nonce for proof events. Must be incremented sequentially.
-    uint256 public state_proofNonce = 1;
+    uint256 public state_proofNonce;
```
Comment (ctian1) [inline contracts/src/BlobstreamX.sol:30]: nit: is there a reason this isn't stateProofNonce or proofNonce?

File: contracts/src/BlobstreamX.sol [hunk 2]
```diff
@@ -41,29 +41,28 @@ contract BlobstreamX is
     /// @notice Next header function id.
     bytes32 public nextHeaderFunctionId;
 
+    struct InitParameters {
```
Comment (ctian1) [inline contracts/src/BlobstreamX.sol:44]: nit: IMO using struct doesn't really help here since its not being passed around

**Review by ctian1 — APPROVED** [review]

File: contracts/src/BlobstreamX.sol (hunk 2 above)
Comment (ratankaliani) [reply contracts/src/BlobstreamX.sol:44]: stack too deep

File: contracts/src/BlobstreamX.sol (hunk 1 above)
Comment (ratankaliani) [reply contracts/src/BlobstreamX.sol:30]: This matches the current variable naming in `Blobstream`. Can definitely change later on, but for now will leave as this.

**Pull request merged**